### PR TITLE
Fix circle precision

### PR DESCRIPTION
**Pull request by nekronos**
This PR contains:
- [x] Changelog
- [ ] Documentation
- [x] Tests

### CODE REVIEW

File: Source/Fuse.Drawing.Primitives/Circle.uno [hunk 1]
```diff
@@ -80,7 +80,8 @@ namespace Fuse.Drawing.Primitives
 		{
 			if (_bufferVertex == null)
 				InitBuffers();
-				
+
+
```
Comment (kusma) [inline Source/Fuse.Drawing.Primitives/Circle.uno]: This part seems pointless

File: Source/Fuse.Drawing.Primitives/Circle.uno [hunk 2]
```diff
@@ -96,8 +97,9 @@ namespace Fuse.Drawing.Primitives
 				VertexCount: _bufferIndex.Count();
 				
 				float2 VertexPosition: V0 * (radius + extend*2);
+				float2 VertexPositionScaled: (V0 * (radius + extend*2)) / radius;
```
Comment (kusma) [inline Source/Fuse.Drawing.Primitives/Circle.uno]: I know I suggested this, but maybe `float2 VertexPositionScaled: VertexPosition / radius;` would be a bit clearer?

File: Source/Fuse.Drawing.Primitives/Circle.uno [hunk 3]
```diff
@@ -96,8 +96,9 @@ namespace Fuse.Drawing.Primitives
 				VertexCount: _bufferIndex.Count();
 				
 				float2 VertexPosition: V0 * (radius + extend*2);
+				float2 VertexPositionScaled: VertexPosition / radius;
```
Comment (yupferris) [inline Source/Fuse.Drawing.Primitives/Circle.uno]: What if radius is 0 ?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+## Circle
+- Fixed issue where Circles could draw incorrect due to floating point precision
```
Comment (yupferris) [inline CHANGELOG.md]: Nit: incorrect -> incorrectly

**Comment by kusma** [comment]
@yupferris: It's not really about precision, it's about range. FP16 will overflow if the circle is too big otherwise.

File: Source/Fuse.Drawing.Primitives/Circle.uno (hunk 3 above)
Comment (kusma) [reply Source/Fuse.Drawing.Primitives/Circle.uno]: Good point. Perhaps earlying out in such cases would be a good move.

**Comment by kusma** [comment]
Also, about a comment, that's probably a good idea. It'd be easy to remove this fix in the future in an effort to "optimize" the code...

**Comment by yupferris** [comment]
> It'd be easy to remove this fix in the future in an effort to "optimize" the code...
right, that's exactly what I thought.

**Pull request merged**